examples/specs/concat_marginal_histograms.vl.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "data": {"url": "data/movies.json"},
+  "spacing": 10,
```

Comment:
@domoritz  Thanks for adding. 

Btw, I'm wondering if 15 is better since the 0 of the top plot is a bit too close to the lower plot right now.

Reply:
I like it being close but I see the argument for more space.

Reply:
I added more spacing and pushed.